types/nodemark/index.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for nodemark 0.3
+// Project: https://github.com/JoshuaWise/nodemark
+// Definitions by: Beeno Tung <https://github.com/beenotung>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace benchmark {
+    type SyncFn = () => void;
+
+    type AsyncFn = (cb: DoneCallback) => void;
+
+    type DoneCallback = () => void;
+
+    type Unit = 'nanoseconds' | 'microseconds' | 'milliseconds' | 'seconds' | 'hz';
+
+    interface BenchmarkResult {
+        mean: number; // the average measured time in nanoseconds
+        error: number; // the margin of error as a ratio of the mean
+        max: number; // the fastest measured time in nanoseconds
+        min: number; // the slowest measured time in nanoseconds
+        count: number; // the number of times the subject was invoked and measured
+
+        nanoseconds(
+            precision?: number, // default to 0
+        ): number;
+
+        microseconds(
+            precision?: number, // default to 0
+        ): number;
+
+        milliseconds(
+            precision?: number, // default to 0
+        ): number;
+
+        seconds(
+            precision?: number, // default to 0
+        ): number;
+
+        hz(
+            precision?: number, // default to 0
+        ): number;
+
+        sd(
+            precision?: number, // default to 0
+        ): number;
+
+        toString(
+            format?: Unit, // default to 'hz'
+        ): string;
+
+        // [util.inspect.custom]: () => string;
```

Comment:
```suggestion
```
no dead code

Reply:
I want to hint that field exists but not sure how to do that within the type system